civictechprojects/views.py

```diff
@@ -110,6 +114,8 @@ def group_edit(request, group_id):
 
 # TODO: Pass csrf token in ajax call so we can check for it
 @csrf_exempt
+@api_view(['GET', 'DELETE'])
```

Comment:
We actually use POST here, when we should have used DELETE, same with the other deletion operations.
```suggestion
@api_view(['GET', 'POST'])
```